circuit/src/poseidon/params/mod.rs

```diff
@@ -4,22 +4,27 @@ pub mod bn254_5x5;
 use halo2wrong::halo2::{arithmetic::FieldExt, plonk::Expression};
 
 pub trait RoundParams<F: FieldExt, const WIDTH: usize>: Sbox {
+	/// Returns integer 8 as usize.
 	fn full_rounds() -> usize;
+	/// Returns integer 60 as usize.
 	fn partial_rounds() -> usize;
 
+	/// Returns total count size.
 	fn round_constants_count() -> usize {
 		let partial_rounds = Self::partial_rounds();
 		let full_rounds = Self::full_rounds();
 		(partial_rounds + full_rounds) * WIDTH
 	}
 
+	/// Returns enough length vector to track counting.
 	fn round_constants() -> Vec<F> {
```

Comment:
This comment is not clear. This function returns the round constants array to be used in permutation.